### PR TITLE
kokoro: Use googltest v1.10.0 in shaderc smoketest

### PR DESCRIPTION
**Pull request by dneto0**

### CODE REVIEW

**Comment by dneto0** [comment]
Doing this as a stopgap until https://github.com/KhronosGroup/SPIRV-Tools/issues/4111 is fixed.  (Also, this pattern can be used for downstream projects like glslang)
Cc: @ben-clayton

File: kokoro/shaderc-smoketest/build.sh [hunk 1]
```diff
@@ -36,7 +36,10 @@ SHADERC_DIR=$GITHUB_DIR/shaderc
 cd $SHADERC_DIR/third_party
 
 # Get shaderc dependencies. Link the appropriate SPIRV-Tools.
-git clone https://github.com/google/googletest.git
+
+# Use v1.10.0 of googltest because it works with older compilers.
+( mkdir googletest && cd googletest && git init && git remote add github https://github.com/google/googletest.git && git fetch github release-1.10.0 && git checkout FETCH_HEAD )
+
```
Comment (s-perron) [inline kokoro/shaderc-smoketest/build.sh:42]: Can't you use `git clone --branch release-1.10.0 https://github.com/google/googletest.git`?

**Review by s-perron — CHANGES_REQUESTED** [review]
I think the command can be a little simpler.  Just add the `--branch` option to the existing git clone command.

File: kokoro/shaderc-smoketest/build.sh (hunk 1 above)
Comment (dneto0) [reply kokoro/shaderc-smoketest/build.sh:42]: Indeed that works!  Thanks

**Comment by dneto0** [comment]
It looks like #4114 is nearing completion, and that is the prefereable path.

**Comment by dneto0** [comment]
#4114 has been merged, so I'll close this

**Pull request closed without merging**